### ISSUE TITLE
Duplicated samples in sample/ and etc.ppp/

### ISSUE THREAD

**Issue by felixonmars** [ppp-project/ppp#520]
Currently we have both sample/options and etc.ppp/options.example, same for the pap-secrets file.

The sample/ versions are far more detailed and suitable for users to look at, IMHO. Would it be a good idea if we take a look and modernize the defaults and ship them instead of the etc.ppp copies?

**Comment by Neustradamus** [comment]
@paulusmack, @enaess: Have you seen this @felixonmars ticket?

**Comment by Neustradamus** [comment]
@paulusmack, @enaess: Have you seen this @felixonmars ticket?

**Comment by Neustradamus** [comment]
@paulusmack, @enaess: Have you seen this @felixonmars ticket?

**Comment by paulusmack** [comment]
> The sample/ copies are far more detailed and suitable for users to look at, IMHO. Would it be a good idea if we take a look and modernize the defaults and ship them instead of the etc.ppp copies?

That sounds like a good idea.  Do you have a concrete proposal?

**Comment by Neustradamus** [comment]
@felixonmars: Have you seen the @paulusmack comment?

**Comment by felixonmars** [comment]
Not really, I don't think I have enough knowledge for that unfortunately...
